baselines/translate/src/v3beta1/translation_service_client.ts.baseline

```diff
@@ -610,9 +610,9 @@ export class TranslationServiceClient {
           protosTypes.google.cloud.translation.v3beta1.IGetGlossaryRequest|undefined,
           {}|undefined>): void;
 /**
- * Gets a glossary. Returns NOT_FOUND, if the glossary doesn't
- * exist.
- *
+    * Gets a glossary. Returns NOT_FOUND, if the glossary doesn&#39;t
```

Comment:
The apostrophe is replaced with '&#39;` everywhere, seems it's not something we want? It might be a result of `safe` being applied.